tests/framework/components/test_manager.py

```diff
@@ -9,7 +9,7 @@
 )
 from vivarium.framework.configuration import build_simulation_configuration
 
-from .mocks import MockComponentA, MockComponentB, MockGenericComponent
+from ...helper_components import MockComponentA, MockComponentB, MockGenericComponent
```

Comment:
Why are you importing with `...` rather than an absolute path?

Reply:
I can never get it to work from the tests/ dir (note that Matt was doing this originally as well fwiw)